utils/build/docker/nodejs/parametric/app.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Do we need to update the node run script in this PR. Can we remove `[java]` from the PR title. This change impacts from than just dd-trace-java. We should run all tests against this PR